test/openssl/test_pkey_ec_group.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+require_relative 'utils'
+
+if defined?(OpenSSL) && defined?(OpenSSL::PKey::EC)
```

Comment:
I know this came from `test_pkey_ec.rb`, but this is redundant. While you are at it, can you update it (and other occurrences in `test_pkey_ec*.rb`) to simply

```suggestion
if defined?(OpenSSL::PKey::EC)
```